.eslintrc.js

```diff
@@ -320,6 +320,12 @@ module.exports = {
         'local/no-forbidden-terms': [2, forbiddenTermsGlobal],
       },
     },
+    {
+      'files': ['**/storybook/*.js'],
```

Comment:
For maintainability, should the storybook entry be removed from the above block and consolidated here?

Reply:
It depends on what we find less important to duplicate. The block above is the only that configures `no-forbidden-terms` to use `forbiddenTermsGlobal`, so it might be valuable as a description of that boundary.

Reply:
The current approach is different blocks for different sets of files. I don't think we should break that precedent.